jabgui/src/main/java/org/jabref/gui/preview/ClipboardContentGenerator.java

```diff
@@ -120,6 +121,13 @@ static ClipboardContent processHtml(List<String> citations) {
         return content;
     }
 
+    static ClipboardContent processMarkdown(List<String> citations) {
+        String result = String.join(CitationStyleOutputFormat.MARKDOWN.getLineSeparator(), citations);
```

Comment:
Where is the convertion to Markdown made?

I think, you need to deal with `setOutputFormat` somehow https://github.com/michel-kraemer/citeproc-java/commit/66cad6b4283158eec6524678c9e5bd887a744cc3